pkg/devfile/parser/sourceAttribute.go

```diff
@@ -7,10 +7,12 @@ import (
 )
 
 const ImportSourceAttribute = "library.devfile.io/imported-from"
+const PluginOverrideAttribute = "library.devfile.io/plugin-override-from"
+const ParentOverrideAttribute = "library.devfile.io/parent-override-from"
```

Comment:
should we keep this as unexported?

Reply:
yes, we can make it unexported. but I'm thinking if we should defined those const in devfile/api?? as we need to check those keys before return an error from validator.  WDYT?

Reply:
I was meaning to ask, how would we check for these in the validation since DWO has their own library; so i dont think we can expect to that. This is will make devfile/api dependant on devfile/library...

If we're sure about the attribute keys and they're the same between devfile/library and DWO library, I think we can declare these in devfile/api

Reply:
yep. That's what I'm mean, we will define those keys in devfile/api.  and I will create another PR  to update to get from devfile/api